mercury/views/simulator.py

```diff
@@ -4,21 +4,70 @@
 
 # from ..forms import VehicleForm, SuspensionForm
 from ..forms import SimulatorForm
+from mercury.models import SimulatedData
 import datetime
 
-# from mercury.models import Vehicle,SuspensionSensor
-
 
+# Modified by: Rajeev
+# User story: #95 Continuous Submission for simulator UI
+# begin
 class SimulatorView(TemplateView):
     template_name = "simulator.html"
     form_class = SimulatorForm
 
     def post(self, request, *args, **kwargs):
-        form = self.form_class(request.POST)
-        if form.is_valid():
-            form.save()
-            return HttpResponseRedirect("/simulator/")
-        return render(request, self.template_name, {"form": form})
+        post_name = request.POST.get("the_name")
+        post_owner = request.POST.get("the_owner")
+        post_created_at = request.POST.get("the_created_at")
+        post_temperature = request.POST.get("the_temperature")
+        post_acceleration_x = request.POST.get("the_acceleration_x")
+        post_acceleration_y = request.POST.get("the_acceleration_y")
+        post_acceleration_z = request.POST.get("the_acceleration_z")
+        post_wheel_speed_fr = request.POST.get("the_wheel_speed_fr")
+        post_wheel_speed_fl = request.POST.get("the_wheel_speed_fl")
+        post_wheel_speed_br = request.POST.get("the_wheel_speed_br")
+        post_wheel_speed_bl = request.POST.get("the_wheel_speed_bl")
+        post_suspension_fr = request.POST.get("the_suspension_fr")
+        post_suspension_fl = request.POST.get("the_suspension_fl")
+        post_suspension_br = request.POST.get("the_suspension_br")
+        post_suspension_bl = request.POST.get("the_suspension_bl")
+        post_initial_fuel = request.POST.get("the_initial_fuel")
+        post_fuel_decrease_rate = request.POST.get("the_fuel_decrease_rate")
+        post_initial_oil = request.POST.get("the_initial_oil")
+        post_oil_decrease_rate = request.POST.get("the_oil_decrease_rate")
+        sim_data = SimulatedData(
```

Comment:
Since you aren't reusing any of the variables from lines 19 to 37, why assign them? You should just set the arguments to the `SimulatedData` object from the `request.POST.get(<blah>)` directly.

Reply:
left it that way so as to make any validations in the coming weeks if needed

Reply:
Remember YAGNI principle (You ain't gonna need it)